snapcraft.yaml

```diff
@@ -101,27 +101,24 @@ parts:
       - usr/share/doc/libmm-glib0
 
   modemmanager:
-    plugin: autotools
+    plugin: meson
     after: [ stack-snaps-tools ]
 
     source: https://git.launchpad.net/ubuntu/+source/modemmanager
     source-type: git
-    source-branch: applied/ubuntu/jammy-updates
+    source-branch: applied/ubuntu/noble
 
-    autotools-configure-parameters:
+    meson-parameters:
       - --prefix=/usr
       - --libdir=/usr/lib
       - --libexecdir=/usr/lib/ModemManager
       - --sysconfdir=/etc
-      - --enable-gtk-doc=no
-      - --with-polkit=no
-      - --enable-vala=no
-      - --disable-maintainer-mode
```

Comment:
Is not this one needed?

Reply:
I do not see this as a valid option when using meson?